tech/database/cassandra/cassandra_installation.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Cassandra
+subsection: cassandra
+section: tech-database
+description: OpenSource database server for high-scale application.
+---
+
+# Apache Cassandra
+
+The Apache Cassandra database is the right choice when you need scalability and
+high availability without compromising performance. Linear scalability and proven
+fault-tolerance on commodity hardware or cloud infrastructure make it the perfect
+platform for mission-critical data.
+
+Cassandra is a partitioned row store. Rows are organized into tables with
+a required primary key. Partitioning means that Cassandra can distribute your
+data across multiple machines in an application-transparent matter. Cassandra
+will automatically re-partition as machines are added/removed from the cluster.
+Row store means that like relational databases, Cassandra organizes data by
+rows and columns. The Cassandra Query Language (CQL) is a close relative of SQL.
+
+## Quick start
+
+Cassandra installation and service initialization on Fedora box:
+
+```bash
+$ sudo dnf install cassandra cassandra-server   # install client/server
+$ sudo systemctl start cassandra                # initialize Cassandra server
```

Comment:
Aren't you duplicating the `installation` page?

Please also check our [writing guide](https://developer-portal.github.io/contributing/write-introduction.html).

I'd avoid the `# comments ` at the end of the line, as not everyone know that they're the comments. Can you move it to a separate line?